render.d.ts

```diff
@@ -0,0 +1,47 @@
+export as namespace render
+
+import { ActionsType, View } from 'hyperapp'
```

Comment:
Does this requires `hyperapp` to be in dependencies or peer/dev deps are enough?